examples/nidmm/continuous-acquire-graph-multiple-points.py

```diff
@@ -204,4 +204,6 @@ def on_close(event):
     error_message = rpc_error.details()
     if rpc_error.code() == grpc.StatusCode.UNAVAILABLE :
         error_message = f"Failed to connect to server on {server_address}"
+    elif rpc_error.code() == grpc.StatusCode.UNIMPLEMENTED:
+        error_message = f"Function not implemented"
     print(f"{error_message}") 
```

Comment:
Can you add a finally block after this and move client.Close from line 197 to there. That will handle closing the session in error scenario cases. We can close only if its a valid session, so maybe something like...
```
finally:
    if('vi' in vars() and vi.id != 0):
        # close the session.
        CheckForError(vi, (client.Close(nidcpower_types.CloseRequest(
            vi = vi
            ))).status)
```

Reply:
made that change